executor/src/witgen/rows.rs

```diff
@@ -193,7 +193,31 @@ impl<T: FieldElement> Debug for Row<'_, T> {
     }
 }
 
-impl<T: FieldElement> Row<'_, T> {
+impl<'a, T: FieldElement> Row<'a, T> {
+    /// Creates a fresh row
```

Comment:
Can you explain what it means for a row to be `fresh`?

Reply:
I added a comment. I would have just called it `new`, but `type Row<'a, T> = WitnessColumnMap<Cell<'a, T>>` and `WitnessColumnMap::new` exists already...